packages/tsc/src/index.ts

```diff
@@ -13,9 +13,33 @@ interface Node {
   next: Set<string>
 }
 
+declare module 'yakumo' {
+  interface Arguments {
+    clean: boolean
+  }
+}
+
 register('tsc', async (project) => {
+  const { targets, argv } = project
+
+  // build clean
+  if (argv.clean) {
+    for (const path in targets) {
+      const fullpath = join(cwd, path)
+      try {
+        const { compilerOptions: { outDir = 'lib' }} = await load(fullpath)
```

Comment:
这里其实有两种情况，一种是 `outDir` 一种是 `outFile`。不过只删除 `outFile` 似乎也不太好，就按你现在的来吧。

Reply:
我觉得还需要考虑 `compilerOptions` 不存在的情况。

```suggestion
        const config = await load(fullpath)
        const outDir = config.compilerOptions?.outDir || 'lib'
```